ush/eva/marine_gdas_adt.yaml

```diff
@@ -0,0 +1,247 @@
+# template YAML file to create EVA YAML files
```

Comment:
It seems that the only difference is the variable name and the bounds between the adt and sst yamls. 
Write a utility that will take a generic yaml (based on what you have here) and will swap the bounds and var name.

Reply:
@guillaumevernieres modify the generic yaml file to swap the bounds and var name.